solidity-verifiers/Cargo.toml

```diff
@@ -18,6 +18,7 @@ revm = {version="3.5.0", default-features=false, features=["std"]}
 rust-crypto = "0.2"
 num-bigint = "0.4.3"
 folding-schemes = { path = "../folding-schemes/"} # without 'light-test' enabled
+noname = { git = "https://github.com/dmpierre/noname" }
```

Comment:
Since this is only used for the example, it should be moved under the `dev-dependencies`.